### ISSUE TITLE
Docs refer to removed systemd-units/kubernetes flags

### ISSUE THREAD

**Issue by hardys** [parca-dev/docs#201]
The docs agent release e.g [here](https://www.parca.dev/docs/systemd) resolves to `v0.10.0-rc.1`, but that release drops support for the `--systemd-units` flag since https://github.com/parca-dev/parca-agent/pull/627 removed it.

```
$ grep -R systemd-units
docs/agent-binary.mdx:sudo parca-agent --node=systemd-test --systemd-units=docker.service --log-level=debug --kubernetes=false --store-address=localhost:7070 --insecure
docs/parca-agent-systemd.mdx:      --systemd-units=SYSTEMD-UNITS,...
docs/parca-agent-systemd.mdx:To profile units, you just need to specify the name of the service in `--systemd-units` flag.
docs/parca-agent-systemd.mdx:+  --systemd-units=docker.service,my-app.service \
docs/systemd.mdx:ExecStart=/usr/bin/parca-agent --http-address=":7071" --node=systemd-test --systemd-units=docker.service,parca.service,parca-agent.service --kubernetes=false --store-address=localhost:7070 --insecure
src/components/HomepageQuickstart.js:./parca-agent --node=systemd-test --systemd-units=parca-agent.service --kubernetes=false`
```

It also looks like `--kubernetes=false ` is no longer valid - should these both just be removed?

**Comment by hardys** [comment]
Looks like this is part of #169 but I'm still not clear if local non-kube usage is expected to work now, any clarification or pointers welcome, thanks! :)

**Comment by metalmatze** [comment]
Good catch. Indeed, we're still working on making `v0.10.0` the latest release. By the time this version gets released the docs should be updated. Right now the docs are still for `v0.9.x`.

**Comment by hardys** [comment]
>Right now the docs are still for v0.9.x.

Thanks for clarifying! Note the code in the docs which generates e.g https://www.parca.dev/docs/systemd#setting-up-parca-agent is picking up v0.10.0-rc.1 already, so it's a little confusing :) 

If there are any pointers in the meantime about how to use the 0.10/system-wide profiling on baremetal that would be welcome!

**Comment by kakkoyun** [comment]
> Thanks for clarifying! Note the code in the docs which generates e.g https://www.parca.dev/docs/systemd#setting-up-parca-agent is picking up v0.10.0-rc.1 already, so it's a little confusing :)

I hadn't realized, this was picking pre-releases. We need to update the docs anyway. Feel free to update the existing flags. At the very least, we will be consistent.
